lib/action.ml

```diff
@@ -62,8 +62,12 @@ module Action (Github_api : Api.Github) (Slack_api : Api.Slack) = struct
     | _ -> []
 
   let partition_issue_comment cfg (n : issue_comment_notification) =
```

Comment:
hard to read, simplify (e.g. this_user_is_ignored is almost same length as the function itself and only used once = not needed)

Reply:
can, will do!